ci-operator/step-registry/rosa/cluster/machinepool/rosa-cluster-machinepool-ref.yaml

```diff
@@ -0,0 +1,40 @@
+ref:
+  as: rosa-cluster-machinepool
```

Comment:
I am writing the step `rosa-conf-machine-pool`.

Reply:
>There is no Infra node for the Rosa hypershift cluster. It is by design.

The data-path tests in classic rosa target infra-nodes, whereas in Rosa-HyperShift we do not have one. Ideally, a hosted-cluster performance should not be any different when we perform data-path tests, however, to have an apples-to-apples comparison, we are placing the target pods on the infra-nodes and performing these tests.

Reply:
There are only worker nodes for the rosa hypershift cluster. There are actually infra-nodes on MC, but it prohibits the regular user to touch it.

Reply:
Yes, the ingress-perf test keeps the router-pods busy and it needs more resource to handle such requests. We usually run same test-config on all platform and compare them for performance degradation, although it is by ROSA Hypershift design we need this for our benchmark testing.